modules/logging-forwarding-azure.adoc

```diff
@@ -32,106 +36,48 @@ data:
 ----
 <1> Must contain a primary or secondary key for the link:https://learn.microsoft.com/en-us/azure/azure-monitor/logs/log-analytics-workspace-overview[Log Analytics workspace] making the request.
 
-To obtain a link:https://learn.microsoft.com/en-us/rest/api/storageservices/authorize-with-shared-key[shared key], you can use this command in Azure CLI:
+. Obtain a link:https://learn.microsoft.com/en-us/rest/api/storageservices/authorize-with-shared-key[shared key], you can use this command in Azure CLI:
 
 [source,text]
 ----
 Get-AzOperationalInsightsWorkspaceSharedKey -ResourceGroupName "<resource_name>" -Name "<workspace_name>”
 ----
 
 
-Create or edit your `ClusterLogForwarder` CR using the template matching your log selection.
+. Create or edit your `ClusterLogForwarder` CR using the template matching your log selection.
 
 .Forward all logs
 [source,yaml]
 ----
-apiVersion: "logging.openshift.io/v1"
-kind: "ClusterLogForwarder"
+apiVersion: observability.openshift.io/v1
+kind: ClusterLogForwarder
 metadata:
-  name: instance
+  name: <log_forwarder_name>
   namespace: openshift-logging
 spec:
+  serviceAccountName: 
+    name: <service_account_name> #<1>
   outputs:
   - name: azure-monitor
     type: azureMonitor
     azureMonitor:
-      customerId: my-customer-id # <1>
-      logType: my_log_type # <2>
-    secret:
-       name: my-secret
-  pipelines:
-  - name: app-pipeline
-    inputRefs:
-    - application
-    outputRefs:
-    - azure-monitor
-----
-<1> Unique identifier for the Log Analytics workspace. Required field.
-<2> link:https://learn.microsoft.com/en-us/azure/azure-monitor/logs/data-collector-api?tabs=powershell#record-type-and-properties[Azure record type] of the data being submitted. May only contain letters, numbers, and underscores (_), and may not exceed 100 characters.
-
-.Forward application and infrastructure logs
-[source,yaml]
-----
-apiVersion: "logging.openshift.io/v1"
-kind: "ClusterLogForwarder"
-metadata:
-  name: instance
-  namespace: openshift-logging
-spec:
-  outputs:
-  - name: azure-monitor-app
-    type: azureMonitor
-    azureMonitor:
-      customerId: my-customer-id
-      logType: application_log # <1>
-    secret:
-      name: my-secret
-  - name: azure-monitor-infra
-    type: azureMonitor
-    azureMonitor:
-      customerId: my-customer-id
-      logType: infra_log #
-    secret:
-      name: my-secret
+      customerId: my-customer-id # <2>
+      logType: my_log_type # <3>
+      azureResourceId: "/subscriptions/111111111" #<4>
```

Comment:
I'd remove these optional fields.   Stick to simple example.   upstream reference doc:   https://github.com/openshift/cluster-logging-operator/blob/master/docs/features/logforwarding/outputs/azure-monitor-log-forwarding.adoc

Reply:
removed optional fields